Answers.md

```diff
@@ -1,2 +1,19 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+1. Describe some of the differences between `.forEach` & `.map`.
+For each iterates over lists and has an operation. Map iterates of a list and transforms it returning another list.
```

Comment:
In JavaScript they iterate over and `map` returns an `Array`, not a `list`. I believe that lists are the equivalent in python. And Objects from JavaScript are called dicts in python. The more you know! 🎉